try:
    print('hello')
    # raise Exception('hello')
except Exception as e:
    print(e)
else:
    print('else')
finally:
    print('finally')

